src/Components/Trending/Trending.scss

```diff
@@ -0,0 +1,59 @@
+.head {
+  text-align: center;
+  color: var(--gdsc-blue-1-100);
+}
+.searchbar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url(./Images/Background.png);
```

Comment:
Same as above. After moving the images in 

`public/images/background.png`, 

change it to 

```css
background-image: url("/images/background.png");
```